components/dashboard/src/teams/TeamUsageBasedBilling.tsx

```diff
@@ -85,6 +87,7 @@ export default function TeamUsageBasedBilling() {
                 window.localStorage.removeItem(`pendingStripeSubscriptionForTeam${team.id}`);
                 clearTimeout(pollStripeSubscriptionTimeout!);
                 setPendingStripeSubscription(undefined);
+                setBillingError(error.message || "Could not subscribe team to Stripe. See console for error messages.");
```

Comment:
Same comment about referring users to the developer console.

Reply:
Alternative proposal:

```suggestion
                setBillingError(`Could not subscribe team to Stripe. ${String(error)}`);
```